src/main/java/com/minecolonies/blocks/AbstractBlockHut.java

```diff
@@ -80,24 +82,32 @@ public TileEntity createNewTileEntity(World world, int meta)
         return new TileEntityColonyBuilding();
     }
 
+    /**
+     * Convert the given metadata into a BlockState for this Block
+     */
+    @NotNull
     @Override
     public IBlockState getStateFromMeta(int meta)
     {
-        EnumFacing facing = EnumFacing.getFront(meta);
-        if (facing.getAxis() == EnumFacing.Axis.Y)
+        EnumFacing enumfacing = EnumFacing.getFront(meta);
+
+        if (enumfacing.getAxis() == EnumFacing.Axis.Y)
         {
-            facing = EnumFacing.NORTH;
+            enumfacing = EnumFacing.NORTH;
         }
-        return this.getDefaultState().withProperty(FACING, facing);
+
+        return this.getDefaultState().withProperty(FACING, enumfacing);
     }
 
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)